openedx/core/djangoapps/content/block_structure/models.py

```diff
@@ -0,0 +1,88 @@
+"""
+Models used by the block structure framework.
+"""
+
+from django.core.files.base import ContentFile
+from django.db import models
+
+from model_utils.models import TimeStampedModel
+from openedx.core.djangoapps.xmodule_django.models import UsageKeyField
+from openedx.core.lib.block_structure.exceptions import BlockStructureNotFound
+
+
+def upload_to_path(bs_model, filename):  # pylint:disable=W0613
```

Comment:
Nit: Could you change this to `# pylint: disable=unused-argument`, for readability?